Userland/Libraries/LibWeb/CSS/Properties.json

```
@@ -1559,6 +1559,17 @@
       "unitless-length"
     ]
   },
+  "object-fit": {
+    "inherited": false,
+    "initial": "fill",
+    "valid-identifiers": [
+      "fill",
+      "contain",
+      "cover",
+      "none",
+      "scale-down"
+    ]
```

Because of the Enums.json entry, this can be:

```suggestion
    "valid-types": [
      "object-fit"
    ]
```